STAT/AliDrawStyle.cxx

```diff
@@ -653,63 +661,52 @@ AliDrawStyle::GetProperty("alirootTestStyle.css","marker_size", "TGraph", "Statu
-  TObjArray *cssStyle = (TObjArray *) AliDrawStyle::GetCssStyle(styleName);
-  TString selectors = "";
-  for(Int_t i = 0; i < cssStyle->GetEntriesFast(); i++){
-    selectors += cssStyle->At(i)->GetName();
-    selectors += " \t";
-  }
-  return selectors;
-}
-/// Function return quantity of objects with specified class from TPad
-/// \param cPad         - name of pad
-/// \param className    - name of class
-/// \return
-Int_t AliDrawStyle::CountObjects(TPad *cPad, TString className){
-  Int_t cnt = 0;
-  for(Int_t c = 0; c < cPad->GetListOfPrimitives()->GetEntries(); c++) if(cPad->GetListOfPrimitives()->At(c)->InheritsFrom(className)) cnt++;
-  return cnt;
+    return actProperty;
 }
-
 ///
 /// \param styleName
 /// \param tempGraph
 /// \param elementName
 /// \param className
 /// \param objName
 /// \param objNum
-void AliDrawStyle::TGraphApplyStyle(const char* styleName, TGraph *tempGraph, TString elementName, TString className, TString objName, Int_t objNum){
-  // if styleName not exist use defaults?
-  if(AliDrawStyle::IsSelected(AliDrawStyle::GetSelector(styleName),elementName, className, objName)){
-    tempGraph->SetMarkerColor(AliDrawStyle::GetNamedIntegerAt(AliDrawStyle::GetProperty(styleName, "marker_color", elementName, className, objName), "", objNum));
-    tempGraph->SetMarkerSize(AliDrawStyle::GetNamedFloatAt(AliDrawStyle::GetProperty(styleName, "marker_size", elementName, className, objName), "", objNum));
-    tempGraph->SetMarkerStyle(AliDrawStyle::GetNamedIntegerAt(AliDrawStyle::GetProperty(styleName, "marker_style", elementName, className, objName), "", objNum));
-    /// lines
-    tempGraph->SetLineColor(AliDrawStyle::GetNamedIntegerAt(AliDrawStyle::GetProperty(styleName, "line_color", elementName, className, objName), "", objNum));
-    tempGraph->SetLineWidth(AliDrawStyle::GetNamedFloatAt(AliDrawStyle::GetProperty(styleName, "line_width", elementName, className, objName), "", objNum));
-    tempGraph->SetLineStyle(AliDrawStyle::GetNamedIntegerAt(AliDrawStyle::GetProperty(styleName, "line_style", elementName, className, objName), "", objNum));
-    /// area
-    tempGraph->SetFillColor(AliDrawStyle::GetNamedIntegerAt(AliDrawStyle::GetProperty(styleName, "fill_color", elementName, className, objName), "", objNum));
-    tempGraph->SetFillStyle(AliDrawStyle::GetNamedIntegerAt(AliDrawStyle::GetProperty(styleName, "fill_style", elementName, className, objName), "", objNum));
-  }
+void AliDrawStyle::TGraphApplyStyle(const char* styleName, TGraph *tempGraph, TString elementName, TString className, TString objName){
+  Int_t objNum = 0;
+  TPRegexp numPat0("[[].*[]]");
+  TPRegexp numPat1("[0-9]+");
+  TPRegexp objPat(".*[[?]|.*[.]class|.*");
+  objNum = TString(TString(objName(numPat0))(numPat1)).Atoi();
+  objName = TString(objName(objPat)).ReplaceAll("[", "").ReplaceAll(".class", "");
+  TString pMarkerColor = AliDrawStyle::GetProperty(styleName, "marker_color", elementName, className, objName);
```

Comment:
Example modification for color:

Bool_t status;
TString pMarkerColor = AliDrawStyle::GetProperty(styleName, "marker_color", elementName, className, objName);
value=AliDrawStyle::GetNamedFloatAt(pMarkerSize, "", objNum,status);
if (status)graph->SetMarkerStyle(value);